test/Kestrel.InMemory.FunctionalTests/Http2/Http2TestBase.cs

```diff
@@ -141,7 +141,7 @@ public Http2TestBase()
 
             _echoApplication = async context =>
             {
-                var buffer = new byte[Http2Frame.MinAllowedMaxFrameSize];
+                var buffer = new byte[Http2Limits.MaxAllowedMaxFrameSize];
```

Comment:
There's a test that I added that needs to echo a data frame that's bigger than the `MinAllowedMaxFrameSize` and I wanted to reuse the _echoApplication. I could write a separate request delegate so that each test doesn't allocate 16 MB though.

Reply:
Nevermind, I forgot I restructured the test in question so this limit increase is no longer needed.